.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,101 @@
+name: Test
```

Comment:
Misleading, since this checks formatting (style), unit tests (sdk-tests), and integration tests (examples). Our other CI definitions just use the language name here, i.e. "Go" or "Golang" and they name the YAML file to match i.e. go.yml

Reply:
Sure, "Presubmit checks" works for me.

Reply:
Done.